nautobot/users/filters.py

```diff
@@ -69,6 +69,28 @@ def search(self, queryset, name, value):
         )
 
 
+class TokenFilterSet(BaseFilterSet):
+    q = django_filters.CharFilter(
+        method="search",
+        label="Search",
+    )
+    user_id = django_filters.ModelMultipleChoiceFilter(
+        field_name="user",
+        queryset=get_user_model().objects.all(),
+        label="User",
+    )
+    user = django_filters.ModelMultipleChoiceFilter(
+        field_name="user__username",
+        queryset=get_user_model().objects.all(),
+        to_field_name="username",
+        label="User (name)",
+    )
+
```

Comment:
```suggestion
class TokenFilterSet(BaseFilterSet):
    q = django_filters.CharFilter(
        method="search",
        label="Search",
    )
```

Reply:
Since this endpoint only pertains to an individual users' token, no need to permit searching.